### ISSUE TITLE
Bug report

### ISSUE THREAD

**Issue by Kristoffermyr** [InZidiuZ/op-framework-issue-tracker#144]
**Reporter:**
Kristoffer#9080

**Summary:**
i am able to use the weapon wheel by using the scroll mouse. if i take up my phone my map shows up aswell. And its not in the video im showing you know but the first time this happend i died when i took the train out at got muged by locals instanly loosing 1.4k worth of stuff. (Advanced lockpick and a phone) some other things aswell but cant remember what it was so if i could get like 1.4k it would be appriciated but if i cant get it i understand since i got no proof 

**Steps to Reproduce:**
No idea how just happens 

**Expected Results:**
You can use your gun without pressing (2). I actully stored my gun aswell but was still able to use it. And when u use your phone the map shows up aswell. And you might die when you take the train out 

## Additional Information
you re not able to crouch you zoom in instead. You can s\use your gun without having it in your inventory. and you might die by taking the train and instantly waking up at the hospital

**Screenshots:**
https://youtu.be/SPDiRjFJiwk

**Game Resolution:**
1920x1080

**Player Data:**
https://cdn.discordapp.com/attachments/584775605208219648/595395265570013221/unknown.png

**Notes:**
If i could get back my lockpick and my phone it would be appriciated. And hopefully its a easy fixable glitch

**Comment by Kristoffermyr** [comment]
it has only happend on server 1 atm

**Comment by InZidiuZ** [comment]
Hey,

I've investigated this, and found out it was caused by something separately, which would cause a loop to break, taking with it different functions involving the removal of the old weapon menu.

It seems like it was caused by the native that would prevent you from getting "cop stars" when shooting, stealing a vehicle, etc. and this native would break at times when on OneSync, probably when actually receiving a star.

Anyways, I found another method of preventing those stars, so implemented that one instead, so after next patch this will hopefully be resolved on the server(s) utilizing OneSync.

**InZidiuZ**
*OP-FW Developer*
